deployment/charts/universalnode/templates/service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
Mental note: if we want the app to be reachable outside the cluster we have to add an ingress